docs/vhost-user-net-testing.md

```diff
@@ -30,34 +33,34 @@ Prior to running the test, the following steps need to be performed.
 - Install DPDK
 - Install OVS
 
-Here are some good references for detailing them.
-- Red Hat
-	* https://wiki.qemu.org/Documentation/vhost-user-ovs-dpdk
-- Ubuntu server
-	* https://help.ubuntu.com/lts/serverguide/DPDK.html
-	* https://software.intel.com/en-us/articles/set-up-open-vswitch-with-dpdk-on-ubuntu-server
+Here is a good reference for setting up OVS with DPDK:
+https://docs.openvswitch.org/en/latest/intro/install/dpdk/.
 
 ## Test
-The test runs with multiple queue (MQ) support enabled, using 2 pairs of TX/RX queues defined for both OVS and the virtual machine. Here are the details on how the test can be run.
+The test runs with multiple queue (MQ) support enabled, using 2 pairs of
+TX/RX queues defined for both OVS and the virtual machine. Here are the
+detailed instructions.
 
 _Setup OVS_
 
 `ovs_test.sh` is created to setup and start OVS. OVS will provide the `dpdkvhostuser` backend running in server mode.
 ```bash
-mkdir -p /var/run/openvswitch
+export OVS_PATH=/usr/local/var/run/openvswitch
+export DB_SOCK=/usr/local/var/run/openvswitch/db.sock
+export CONFIG_PATH=/usr/local/etc/openvswitch/conf.db
+mkdir -p $OVS_PATH
 modprobe openvswitch
 killall ovsdb-server ovs-vswitchd
-rm -f /var/run/openvswitch/vhost-user*
-rm -f /etc/openvswitch/conf.db
-export DB_SOCK=/var/run/openvswitch/db.sock
-ovsdb-tool create /etc/openvswitch/conf.db /usr/share/openvswitch/vswitch.ovsschema
+rm -f $OVS_PATH/vhost-user*
+rm -f $CONFIG_PATH
+ovsdb-tool create $CONFIG_PATH /usr/local/share/openvswitch/vswitch.ovsschema
```

Comment:
These instructions are a bit heavy from what I've seen. I followed the instructions here: https://ubuntu.com/server/docs/openvswitch-dpdk and it was working fine for me.
Particularly, I find that we don't need to setup the database for such simple use case.
I mean the point is to simply setup an OVS bridge with two ports, and then we want to validate that the 2 VMs can communicate together. I think you can shrink these instructions (especially the ones tweaking cpu mask, etc..) in order to make them easy to understand :)

Reply:
@sboeuf That's a really good pointer. I took that as a reference and understood better the way how `ovs-vsctl` works. I updated the documents and I think it is now cleaner (and simpler). Please let me know whether you believe it can further simplified (particularly on the `ovs-vsctl set Open_vSwitch ...` part). Any other comments, please let me know. Thank you!